.circleci/config.yml

```diff
@@ -181,6 +194,19 @@ workflows:
           requires:
             - build_and_test
       - perf_test:
+          name: perf-test-flux
```

Comment:
Are we going to run the flux perf tests on 1.8?

Reply:
I think no, will remove this